src/Type/BaseStatusType.php

```diff
@@ -19,31 +19,14 @@
 
 abstract class BaseStatusType extends AbstractType
 {
-    /**
-     * @var string
-     *
-     * @phpstan-var class-string
-     */
-    protected $class;
```

Comment:
This could also cause bc breaks if child clases redeclare protected $class, since now it is protected string $class.